### ISSUE TITLE
BungeeCord texture packs

### ISSUE THREAD

**Issue by john01dav** [SpigotMC/BungeeCord#1226]
Migrated from: http://www.spigotmc.org/threads/bungeecord-texture-packs.30286/

Hello Spigot! I would like to request that you add a resource pack option to config.yml of BungeeCord. This would function exactly the same as the option on Spigot's server.properties. Except, that it will be for the entire network. Right now, I have it set up where the texture pack is given when the user joins the lobby, but this causes freezes when they join the lobby from another server.

**Comment by Zeryther** [comment]
Agree! I'll be running a server network with a custom texturepack soon and that would be really useful.

**Comment by x9nico** [comment]
Really useful.
